contracts/src/bridge/SequencerInbox.sol

```diff
@@ -133,6 +138,13 @@ contract SequencerInbox is DelegateCallAware, GasRefundEnabled, ISequencerInbox
         if (msg.sender != tx.origin) revert NotOrigin();
         if (!isBatchPoster[msg.sender]) revert NotBatchPoster();
         if (inboxAccs.length != sequenceNumber) revert BadSequencerNumber();
+        bytes32 dasKeysetHash = dasKeysetHashFromBatchData(data);
+        if (dasKeysetHash != bytes32(0)) {
+            if (
+                dasKeysetManager == DasKeysetManager(address(0)) ||
```

Comment:
I think it might be better to leave out the `dasKeysetManager == DasKeysetManager(address(0))` check as a guard against configuration issues since then we'll just never accept a DAS message unless the keyset manager is configured